.github/workflows/test.yml

```
@@ -19,7 +19,7 @@ env:
 jobs:
   run:
     strategy:
-      fail-fast: true
+      fail-fast: false
```

```suggestion
      fail-fast: true
```
Reverting this back before merging, right?